src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,22 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../types/Todo';
+import { User } from '../types/User';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user = todo.user as User;
```

Comment:
Casting `todo.user` to `User` using `as User` can be risky if `todo.user` is `null`. Consider checking if `todo.user` is not `null` before casting, or handle the case where `todo.user` might be `null` to ensure type safety.